app/views/products/_product.json.jbuilder

```diff
@@ -1,2 +1,2 @@
-json.extract! product, :id, :title, :description, :image_url, :price, :created_at, :updated_at
-json.url product_url(product, format: :json)
+json.title product.title
```

Comment:
what does this file do, what happens if I delete this file?

Reply:
Deleting this file will throw Missing template error but if we are called to_json as i have in the updated approach then this file can be deleted and no error will be thrown